builtin/rebase.c

```diff
@@ -27,29 +35,51 @@ static int use_builtin_rebase(void)
 	strbuf_release(&out);
 	return ret;
 }
-static int finish_rebase(){
-
+static int finish_rebase(void)
+{
+	const struct object_id *new_head;
+	struct strbuf reflog_message = STRBUF_INIT;
+	struct commit *head_commit;
+	const struct object_id *head = &head_commit->object.oid;
+	unlink("$(git rev-parse --git-path REBASE_HEAD)"); //NEEDS HELP.
+	//apply_autostash();
+	const char *argv_gc_auto[] = { "gc", "--auto", NULL };
+			update_ref(reflog_message.buf, "HEAD", new_head, head,
+				   0, UPDATE_REFS_DIE_ON_ERR);
+			/*
+			 * We ignore errors in 'gc --auto', since the
+			 * user should see them.
+			 */
+			close_all_packs(the_repository->objects);
+			run_command_v_opt(argv_gc_auto, RUN_GIT_CMD);
+	remove_dir_recursively(git_path_rebase_apply());
+	remove_dir_recursively(git_path_rebase_merge());
```

Comment:
remove_dir_recursively() takes 2 arguments, so I don't see how this can even compile.